lib/vm/src/trap/traphandlers.rs

```diff
@@ -546,6 +546,9 @@ pub unsafe fn wasmer_call_trampoline(
     callee: *const VMFunctionBody,
     values_vec: *mut u8,
 ) -> Result<(), Trap> {
+    // `vmctx` is always `*mut VMContext` here, as we call to WASM.
+    let ctx = vmctx.vmctx;
+    (ctx.as_ref().unwrap()).instance().on_call();
```

Comment:
```suggestion
    (*ctx).instance().on_call();
```

`.unwrap()` would generate "exception throwing" machinery, which we probably want to avoid in this low-level function.